mozilla_django_oidc_db/checks.py

```diff
@@ -0,0 +1,93 @@
+import inspect
+from collections.abc import Sequence
+from typing import Any
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.core.checks import CheckMessage, Error, Warning, register
+from django.utils.module_loading import import_string
+
+from .views import OIDCCallbackView, OIDCInit
+
+
+def _do_check(
```

Comment:
I tested this with openforms, but I can't get the checks to work? It seems that the actual imports are run before these checks.

`base.py`:
```python
OIDC_AUTHENTICATE_CLASS = "foo"
```

```
File "env/lib/python3.10/site-packages/mozilla_django_oidc/urls.py", line 20, in <module>
    OIDCAuthenticateClass = import_string(AUTHENTICATE_CLASS_PATH)
  File "env/lib/python3.10/site-packages/django/utils/module_loading.py", line 27, in import_string
    raise ImportError("%s doesn't look like a module path" % dotted_path) from err
ImportError: foo doesn't look like a module path
```

Reply:
yeah I don't test for the importability because of this reason - the modules are imported at django startup time because the urlconf evaluates them (in the upstream library).

Try setting it to something that can be imported, like `django.views.View`.

Reply:
Aah okay, then it works as expected indeed :+1: